BFWControls/Modules/NibView/View/NibTextField.swift

```diff
@@ -112,10 +112,8 @@ open class NibTextField: UITextField {
 private extension UIView {
     
     var subviewTextField: UITextField? {
-        return subviews.first
-            { subview in
-                subview is UITextField || subview.subviewTextField != nil
-            } as? UITextField
+        let textFields = subviews.flatMap { $0 as? UITextField ?? $0.subviewTextField }
+        return textFields.first
     }
```

Comment:
It is bit hard to use ```subviews.first {...}``` in this scenario because we need to traverse all subviews recursively to find UITextField but ```subviews.first {...}``` can't traverse into next array.  Can I use this for now and refactor it later today?